infra/src/main/java/com/redhat/service/smartevents/infra/exceptions/mappers/ConstraintViolationExceptionMapper.java

```diff
@@ -0,0 +1,88 @@
+package com.redhat.service.smartevents.infra.exceptions.mappers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Function;
+
+import javax.inject.Inject;
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.ResponseBuilder;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.hibernate.validator.engine.HibernateConstraintViolation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.smartevents.infra.api.models.responses.ErrorResponse;
+import com.redhat.service.smartevents.infra.api.models.responses.ErrorsResponse;
+import com.redhat.service.smartevents.infra.exceptions.BridgeError;
+import com.redhat.service.smartevents.infra.exceptions.BridgeErrorService;
+import com.redhat.service.smartevents.infra.exceptions.definitions.platform.UnclassifiedConstraintViolationException;
+import com.redhat.service.smartevents.infra.exceptions.definitions.user.ExternalUserException;
+import com.redhat.service.smartevents.infra.models.ListResult;
+
+public class ConstraintViolationExceptionMapper implements ExceptionMapper<ConstraintViolationException> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConstraintViolationExceptionMapper.class);
+
+    private final ErrorResponseConverter converter = new ErrorResponseConverter();
+
+    @Inject
+    BridgeErrorService bridgeErrorService;
+
+    @Override
+    public Response toResponse(ConstraintViolationException e) {
+        LOGGER.debug("Failure", e);
```

Comment:
What is the purpose of this logging entry? What should be tracked/stored into log in here?

When I print this log into console it doesn't give me much information:
```
com.redhat.service.smartevents.manager.api.models.requests.ProcessorRequest@4ddca75f]nMapper] (executor-thread-0) Failure: [PARAMETER]
	at io.quarkus.hibernate.validator.runtime.jaxrs.JaxrsEndPointValidationInterceptor.validateMethodInvocation(JaxrsEndPointValidationInterceptor.java:37)
	at io.quarkus.hibernate.validator.runtime.jaxrs.JaxrsEndPointValidationInterceptor_Bean.intercept(Unknown Source)
	at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:41)
```